test/suppCodec.mpd

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
```

Comment:
Either it's linked to a real test, in which case it would be stored in `test/contents/` along a test case in `test/integration` either we may want to unstage this.